src/mono/mono/metadata/qcall-def.h

```diff
@@ -0,0 +1,3 @@
+#ifndef NO_GLOBALIZATION_SHIM
```

Comment:
Please add a header comment to this file that:
1. Says what it is for
2. Gives a "declaration" that describes `FCClassElement` (something like `FCClassElement (class_name, namespace, symbol_name)` where `symbol_name` is an array of `MonoQCallFunc`.
3. Includes a note that the `FCClassElements` in the header have to be sorted by name then namespace, and that the functions in each one have to be sorted by name.